cmd/internal/cli/push.go

```diff
@@ -24,6 +32,9 @@ func init() {
 	PushCmd.Flags().StringVar(&PushLibraryURI, "library", "https://library.sylabs.io", "the library to push to")
 	PushCmd.Flags().SetAnnotation("library", "envkey", []string{"LIBRARY"})
 
+	PushCmd.Flags().BoolVarP(&unauthenticatedPush, "allow-unauthenticated", "U", false, "dont check if the container is signed")
+	PushCmd.Flags().SetAnnotation("allow-unauthenticated", "envkey", []string{"ALLOW_UNAUTHENTICATED"})
```

Comment:
Should this be `ALLOW_UNAUTHENTICATED`? or `SINGULARITY_ALLOW_UNAUTHENTICATED`?

Reply:
`ALLOW_UNAUTHENTICATED`, this is just specifying the suffix of the environment variable that will need to be set